### PR TITLE
UI for rasterization operator

### PR DESCRIPTION
**Pull request by arianegr**
New rasterization component

### CODE REVIEW

**Comment by arianegr** [comment]
[[GFBio] Rasterization für Punktdaten (Heatmap)](https://trello.com/c/ajTGM5DC/994-gfbio-rasterization-f%C3%BCr-punktdaten-heatmap)

File: projects/core/src/lib/operators/dialogs/rasterization/rasterization.component.html [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+                    <mat-radio-button value="relative">Relative</mat-radio-button>
+                </mat-radio-group>
+                <div formGroupName="resolution" class="groupInRow">
+                    <mat-form-field appearance="fill">
+                        <mat-label>Resolution (x)</mat-label>
+                        <input matInput type="number" formControlName="resX" />
+                    </mat-form-field>
+                    <mat-form-field appearance="fill">
+                        <mat-label>Resolution (y)</mat-label>
+                        <input matInput type="number" formControlName="resY" />
```
Comment (ChristianBeilschmidt) [inline projects/core/src/lib/operators/dialogs/rasterization/rasterization.component.html:43]: it shouldn't be allowed to have zero or negative resolutions

File: projects/core/src/lib/operators/dialogs/rasterization/rasterization.component.html [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+            <ng-template #densityTemplate>
+                <div class="groupInRow" *ngIf="selected.value === 1">
+                    <mat-form-field appearance="fill">
+                        <mat-label>Radius</mat-label>
+                        <input matInput type="number" formControlName="radius" />
+                    </mat-form-field>
+                    <mat-form-field appearance="fill">
+                        <mat-label>Stddev</mat-label>
+                        <input matInput type="number" formControlName="stddev" />
+                    </mat-form-field>
```
Comment (ChristianBeilschmidt) [inline projects/core/src/lib/operators/dialogs/rasterization/rasterization.component.html:66]: These values should be positive.

Moreover, if there is the possibility to set the operator wrongly, clicking create does not result in any notification to the user that this failed.

**Review by ChristianBeilschmidt — CHANGES_REQUESTED** [review]

File: projects/core/src/lib/operators/dialogs/rasterization/rasterization.component.html (hunk 1 above)
Comment (arianegr) [reply projects/core/src/lib/operators/dialogs/rasterization/rasterization.component.html]: da68e10

File: projects/core/src/lib/operators/dialogs/rasterization/rasterization.component.html (hunk 2 above)
Comment (arianegr) [reply projects/core/src/lib/operators/dialogs/rasterization/rasterization.component.html]: 6f87f00

**Pull request merged**